timemachine/cpp/src/nonbonded_all_pairs.cu

```diff
@@ -31,18 +31,18 @@ NonbondedAllPairs<RealType>::NonbondedAllPairs(
       d_sort_storage_bytes_(0), disable_hilbert_(disable_hilbert_sort),
 
       kernel_ptrs_({// enumerate over every possible kernel combination
+                    // Set threads to 1 if not computing energy to reduced unused shared memory
```

Comment:
Can we set this to zero?

Reply:
Sadly we can not

Reply:
Offline discussion, this gets removed by the compiler if its unused, so made consistent for visual purposes.